src/types/error.ts

```diff
@@ -27,13 +37,42 @@ export function notFoundError(message: string): ServiceErrorNotFound {
   return { message, kind: "ServiceErrorNotFound" } as ServiceErrorNotFound;
 }
 
+export function forbiddenError(message: string): ServiceErrorGeneric {
+  return { message, kind: "ServiceErrorGeneric" } as ServiceErrorGeneric;
+}
+
+const ErrorGenericBody = t.interface({
+  detail: t.string,
+  status: t.number,
+  title: t.string,
+  type: t.string
+});
+type ErrorGenericBody = t.TypeOf<typeof ErrorGenericBody>;
+
 export function toHttpError(
   err: ServiceError
-): IResponseErrorNotFound | IResponseErrorInternal {
+): IResponseErrorNotFound | IResponseErrorInternal | IResponseErrorGeneric {
   switch (err.kind) {
     case "ServiceErrorInternal":
       return ResponseErrorInternal(err.message);
     case "ServiceErrorNotFound":
       return ResponseErrorNotFound("Not found", err.message);
+    case "ServiceErrorGeneric":
+      // Forbidden Error: Message into body should be propagated as is
+      const errorGenericBodyOrError = ErrorGenericBody.decode(err.message);
+      if (
+        isRight(errorGenericBodyOrError) &&
+        errorGenericBodyOrError.value.status === 403
```

Comment:
don't use `ServiceErrorGeneric`, we do have a Forbidden error response: https://github.com/teamdigitale/italia-ts-commons/blob/master/src/responses.ts#L241

Reply:
ResponseErrorForbiddenNotAuthorized is defined as a default readonly error (obtained throw ResponseErrorGeneric):
```
...ResponseErrorGeneric(
    HttpStatusCodeEnum.HTTP_STATUS_403,
    "You are not allowed here",
    "You do not have enough permission to complete the operation you requested"
)
```

I need to define a custom Forbidden error setting a custom message into body. For this reason, I'm defining it call directly ResponseErrorGeneric constructor and passing parameters to him.
In this way, information provided by PagoPa will be propagated as is.

Reply:
Actually, we don't want to send the errors coming from pagopa-rpxy to the app, we want to log them in the backend and send to the app a generic 403 error - this limits potential information leakage to malicious actors. Please use the provided Forbidden error. (you can see also the other errors don't leak information coming from the API backend to the app).

Reply:
Ok, I'm going to change it.

@fgiobergia 
This means that APP will not be able to check error reasons as described here:
https://docs.google.com/spreadsheets/d/1n1PLKredBY0OH7HzBerAC-xWJwlu9jPkTS9moMKl8CY/edit#gid=0

Reply:
@LsKoder what kind of errors are we talking about here? because 403 is very specific!

For more applicative errors you may use `ResponseErrorInternal`.

Reply:
I'm talking about errors mapped into the file I specified (please see the first column).
They are actually mapped as 403 on PagoPaProxy.

Reply:
`PAYMENT_UNAVAILABLE` ? I don't think it makes sense to return a 403 (neither by the pagopa proxy) as 403 means the client is not authorized (as in doesn't have the required permission) to issue the request.

If you want to map all those errors into a single HTTP status code, I suggest to use 500 that is suitable fo a generic error.

Reply:
Ok, let's cancel this PR. Any application errors provided by proxypagopa will be mapped on 500.
I'm going to make a PR on PagoPaProxy project.